### PR TITLE
make divine blade simple, add to sorcerer

### PR DESCRIPTION
**Pull request by Silvergriffon**
Created a "ConjuredSwordType" weapon type specifically for the Divine Blade spell's summoned blade. Copied longsword, but made it simple, then added it to sorcerer so that Divine Heart subclass can wield it too. If you would rather wait until after TA'
s next patch in case they fix it, then deny this and I will resubmit if necessary.

### CODE REVIEW

File: SolastaUnfinishedBusiness/Models/FixesContext.cs [hunk 1]
```diff
@@ -85,6 +86,23 @@ private static void FixColorTables()
...
+            .SetWeaponCategory(WeaponCategoryDefinitions.SimpleWeaponCategory)
+            .AddToDB();
+
+        var divineBladeWeaponDefinition = ItemDefinitions.DivineBladeWeapon.weaponDefinition;
+        divineBladeWeaponDefinition.weaponType = "ConjuredSwordType";
+
+        ItemDefinitions.DivineBladeWeapon.weaponDefinition = divineBladeWeaponDefinition;
+
+        //BUGFIX: allows divine heart sorcerer to wield divine blade
+        FeatureDefinitionProficiencys.ProficiencySorcererWeapon.proficiencies.Add("ConjuredSwordType");
```
Comment (ThyWoof) [inline SolastaUnfinishedBusiness/Models/FixesContext.cs:103]: can you also add to Druid and Wizard?

**Review by ThyWoof — APPROVED** [review]

**Comment by ThyWoof** [comment]
dev_future has the latest beta dump and still no changes there.

File: SolastaUnfinishedBusiness/Models/FixesContext.cs (hunk 1 above)
Comment (dolenam317) [reply SolastaUnfinishedBusiness/Models/FixesContext.cs:103]: I didnt see any bug report yet. Does TA aware of this ?

Comment (ThyWoof) [reply SolastaUnfinishedBusiness/Models/FixesContext.cs:103]: I did on internal channel

**Pull request merged**